integration-test/run-it.sh

```diff
@@ -91,6 +90,11 @@ function start_storm_process() {
     echo starting: storm $1
     sudo su storm -c "export JAVA_HOME=\"${JAVA_HOME}\" && cd /usr/share/storm && storm $1" &
 }
+if [[ "${USER}" == 'travis' ]]; then
+    start_storm_process dev-zookeeper
```

Comment:
I see what appears to be a Vagrant use case, and wanted to scope my change to Travis in case others depended on using a packaged ZK.

We can change the script to do this in any case. I just didn't know what the expectations were. In the vagrant case, the version of the packages is even different (3.4.5 versus 3.4.8).

But I'd be happy to simplify this further if we want.

Reply:
OK this is done.